<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MusicItemwi</class>
 <widget class="QWidget" name="MusicItemwi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>80</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1170</width>
    <height>80</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1170</width>
    <height>80</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border: none;
background-color: transparent;
border-radius: 15px;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widgetMusicItem_wi" native="true">
     <property name="minimumSize">
      <size>
       <width>1150</width>
       <height>80</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1150</width>
       <height>80</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#widgetMusicItem_wi {
	border-radius: 15px;
}
#widgetMusicItem_wi:hover{
	background-color: #FFFFFF;
}</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <spacer name="horizontalSpacer_7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>5</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="index_btn">
        <property name="minimumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
         font-weight: 500;
        background-color: transparent;</string>
        </property>
        <property name="text">
         <string>01</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>15</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>340</width>
          <height>60</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>340</width>
          <height>60</height>
         </size>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="songname_lb">
           <property name="styleSheet">
            <string notr="true">color: #283248;
        font-size: 21px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;</string>
           </property>
           <property name="text">
            <string>歌曲名</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="composer_lb">
           <property name="styleSheet">
            <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;</string>
           </property>
           <property name="text">
            <string>作者</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>176</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="album_lb">
        <property name="minimumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #283248;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;</string>
        </property>
        <property name="text">
         <string>专辑</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="duration_lb">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #8b949e;
        font-size: 20px;
        font-family: &quot;Microsoft YaHei&quot;;
     font-weight: 500;
        background-color: transparent;</string>
        </property>
        <property name="text">
         <string>时长</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>45</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="size_lb">
        <property name="minimumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: #8b949e;
        font-size: 19px;
        font-family: &quot;Microsoft YaHei&quot;;
        font-weight: normal;
        background-color: transparent;</string>
        </property>
        <property name="text">
         <string>大小</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Maximum</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
